<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Call Tree Studio</string>
  </property>
  <widget class="QWidget" name="m_CentralWidget">
   <widget class="QLabel" name="m_StatusLabelCurrentFile">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="indent">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="m_Menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_MenuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_ActionOpen"/>
   </widget>
   <widget class="QMenu" name="m_MenuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_ActionAboutQt"/>
   </widget>
   <addaction name="m_MenuFile"/>
   <addaction name="m_MenuHelp"/>
  </widget>
  <widget class="QStatusBar" name="m_Statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="m_ActionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_ActionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
